rust-code-analysis-cli/src/main.rs

```diff
@@ -234,6 +252,7 @@ fn main() {
                 .long("metrics")
                 .short('m'),
         )
+        .arg(Arg::new("wmc").help("Compute WMC").long("wmc"))
```

Comment:
I know what `WMC` means thanks to the PR title but in two days I'll have forgotten.
So please, add something to explain what it is.

Reply:
I added the full metric name to the command so that it shows up when invoking `--help`